e2e/fixtures/fixtures.go

```diff
@@ -92,8 +92,8 @@ func NewService(app, host, keyvaultURI string, port int32) *corev1.Service {
 		Spec: corev1.ServiceSpec{
 			Ports: []corev1.ServicePort{{
 				Name:       "http",
-				Port:       port,
-				TargetPort: intstr.FromInt(8080),
+				Port:       1234, // anything - we don't use this one
+				TargetPort: intstr.FromInt(int(port)),
```

Comment:
Start using the correct port in e2e - previously we used Port for the upstream connections which is wrong